presto-parser/src/main/java/com/facebook/presto/sql/TreePrinter.java

```diff
@@ -115,9 +119,23 @@ protected Void visitQuerySpecification(QuerySpecification node, Integer indentLe
                 }
 
                 if (!node.getGroupBy().isEmpty()) {
-                    print(indentLevel, "GroupBy");
-                    for (Expression expression : node.getGroupBy()) {
-                        process(expression, indentLevel + 1);
+                    for (GroupBySpecification groupBySpecification : node.getGroupBy()) {
```

Comment:
This class seems dead. Remove this class in a separate commit (can be same pull request).